mass_mailing_partner/models/res_partner.py

```diff
@@ -40,7 +40,7 @@ def _check_email_mass_mailing_contacts(self):
     def _compute_mass_mailing_contacts_count(self):
         for reg in self:
             reg.mass_mailing_contacts_count = len(
-                reg.mass_mailing_contact_ids)
+                reg.sudo().mass_mailing_contact_ids)
```

Comment:
This one shouldn't be needed

Reply:
Yes, but permissions were raising without it :/